modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -683,6 +683,71 @@ definition {
 			assigneeType = "Organizations");
 	}
 
+	@description = "This test asserts that an Account Admin can unassign roles to users."
+	@priority = "5"
+	test AccountAdminCanUnassignRolesToUsers {
+		task ("Given an Account User is given a role by the Account Administrator") {
+			JSONAccountEntryUser.addAccountEntryUser(
+				accountEntryName = "Account Name",
+				userEmailAddress = "user1@liferay.com",
+				userFirstName = "user1",
+				userLastName = "user1",
+				userScreenName = "user1");
+
+			LexiconEntry.gotoEntry(rowEntry = "Account Name");
+
+			Navigator.gotoNavTab(navTab = "Users");
+
+			Accounts.goToEntryMenuItem(
```

Comment:
Can we move the Assign Roles step to a macro, so we can reuse it in other tests like `AccountAdminCanAssignRolesToUsers`.